[ADD] new doc for release report (#1272)
mercadopago/devsite-docs: guides/manage-account/reports/released-money/api.pt.md

# Geração via API

> Gere o relatório de Liquidações manualmente quantas vezes quiser ou programe-o de acordo com a frequência desejada por meio de nossa API.

Reviewer: ```suggestion

```

> Gere o relatório de Liquidações manualmente quantas vezes quiser ou programe-o de acordo com a frequência desejada por meio de nossa API.

Reviewer: ```suggestion
Gere o relatório de ----[mla]----Liquidações------------ ----[mlm, mlb, mlc, mco, mlu, mpe]----Liberações------------ manualmente quantas vezes quiser ou programe-o de acordo com a frequência desejada por meio de nossa API.
```

> Gere o relatório de Dinheiro disponível manualmente quantas vezes quiser ou programe-o de acordo com a frequência desejada por meio de nossa API.

Reviewer: ```suggestion
```

> WARNING
>

> > O relatório de Dinheiro disponível será desabilitado a partir de 1o de março de 2022.

Reviewer: ```suggestion
> A partir do dia 1º de março de 2022, o relatório Dinheiro disponível será desabilitado
```

>
> Use o [relatório de ----[mla]----Liquidações------------ ----[mlm, mlb, mlc, mco, mlu, mpe]----Liberações------------](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/manage-account/reports/released-money/introduction) para fazer a reconciliação das transações que afetem o saldo disponível na sua conta, incluindo seus saques bancários.

> ## Configurar os seus relatórios

Reviewer: ```suggestion
## Configure seus relatórios
```

> NOTE
>
> Nota
>
> Tenha em mãos o [Glossário](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/manage-account/reports/released-money/glossary) para conferir termos técnicos durante a geração do seu relatório.
Execute o _curl_ que você precisa para consultar, gerar e atualizar os seus relatórios.

> ### Consultando configurações

Reviewer: ```suggestion
### Consulte seus configurações
```

```
```node
var request = require('request');

var headers = {
    'accept': 'application/json',
    'content-type': 'application/json',
    'Authorization': 'Bearer ENV_ACCESS_TOKEN'
};

var options = {
    url: 'https://api.mercadopago.com/v1/account/release_report/config',
    headers: headers
};
function callback(error, response, body) {
    if (!error && response.statusCode == 200) {
        console.log(body);
    }
}
request(options, callback);
```
]]]

Você receberá um `HTTP STATUS 200 (Ok)` como resposta se nenhum erro for encontrado.

O objeto de resposta terá uma estrutura semelhante ao exemplo abaixo:

```json
{
    "file_name_prefix": "release-report-USER_ID",
    "include_withdrawal_at_end": true,
    "scheduled": false,
    "execute_after_withdrawal": false,
    "separator": ";",
    "display_timezone": "GMT-04",
    "frequency": {
        "hour": 0,
        "type": "monthly",
        "value": 1
    },
    "columns": [
        {
            "key": "DATE"
        },
        {
            "key": "SOURCE_ID"
        },
        {
            "key": "EXTERNAL_REFERENCE"
        }
    ]
}
```

> ### Criando configurações

Reviewer: ```suggestion
### Crie as configurações
```

> Crie as suas preferências de geração de relatório via API para exportar colunas, nomear seus arquivos e configurar outros ajustes:

Reviewer: ```suggestion
Crie suas preferências de geração de relatório via API para exportar colunas, nomear seus arquivos e configurar outros ajustes:
```

            "execute_after_withdrawal": falsre,
            "display_timezone": "GMT-04",
            "notification_email_list": [
                "example@email.com",
                "john@example.com"
            ],
            "frequency": {
                "hour": 0,
                "type": "monthly",
                "value": 1
            },
            "columns": [
                {
                    "key": "DATE"
                },
                {
                    "key": "SOURCE_ID"
                },
                {
                    "key": "EXTERNAL_REFERENCE"
                }
            ]
    }';

var options = {
    url: 'https://api.mercadopago.com/v1/account/release_report/config',
    method: 'PUT',
    headers: headers,
    body: dataString
};

function callback(error, response, body) {
    if (!error && response.statusCode == 200) {
        console.log(body);
    }
}

request(options, callback);
```
]]]

Você receberá um `HTTP STATUS 200 (Ok)` como resposta se nenhum erro for encontrado.

O objeto de resposta terá uma estrutura semelhante ao exemplo abaixo:

```json
{
    "file_name_prefix": "release-report-USER_ID",

>     "include_withdrawal_at_end": triue,

Reviewer: ```suggestion
    "include_withdrawal_at_end": true,
```

| `display_timezone` (opcional) | <br/>Este campo determina a data e o horário mostrados nos relatórios. Se você não configurar um fuso horário para esse campo, o sistema considerará o fuso GMT-04 como padrão. Caso escolha um fuso que adote horário de verão, você precisará fazer o ajuste manual quando o horário mudar.<br/><br/> |
| `notification_email_list` (opcional) | <br/>Permite adicionar um grupo de destinatários de e-mail para que recebam uma notificação quando um relatório estiver pronto e disponível para download. Certifique-se de incluir o e-mail associado à sua conta Mercado Pago para que você também receba as notificações.<br/><br/> |
| `refund_detailed` (opcional) | <br/>Mostra o código de referência (external_reference) do reembolso em vez do código de referência (external_reference) do pagamento.<br/><br/> |
| `include_withdrawal` (opcional) | <br/>Inclui as retiradas de dinheiro no relatório.<br/><br/> |
| `coupon_detailed` (opcional) | <br/>Inclui uma coluna para mostrar os detalhes dos cupons de desconto.<br/><br/> |
| `scheduled` (read_only) | <br/>Campo informativo que indica se já existem relatórios programados na conta do usuário.<br/><br/> |

> ## Gerando os seus relatórios manualmente 

Reviewer: ```suggestion
## Gerando seus relatórios manualmente 
```

Gere seus relatórios de forma manual configurando três instâncias: criação, consulta e *download*.

### 1. Criação

> Faça o POST à API especificando as datas de início e fim desejadas, assim:

Reviewer: ```suggestion
Faça uma chamada POST à API especificando as datas de início e fim desejadas, assim:
```

connection.setRequestMethod("GET");
connection.setRequestProperty("Authorization", "Bearer ENV_ACCESS_TOKEN");

System.out.println(connection.getResponseCode());
System.out.println(connection.getResponseMessage());
System.out.println(connection.getInputStream());
```
```python
import requests

headers = { 
    'accept': 'application/json', 
    'Authorization': 'Bearer ENV_ACCESS_TOKEN'
}

response = requests.post('https://api.mercadopago.com/v1/account/release_report/list', headers=headers)
```
```node
var request = require('request');
var headers = { 'accept': 'application/json'};
var dataString = 'access_token=ENV_ACCESS_TOKEN';
var options = {
    url: 'https://api.mercadopago.com/v1/account/release_report/list',
    method: 'POST',
    headers: headers,
    body: dataString
};
function callback(error, response, body) {
    if (!error && response.statusCode == 200) {
        console.log(body);
    }
}
request(options, callback);
```
]]]

Você receberá um `HTTP STATUS 200 (Ok)` como resposta se nenhum erro for encontrado.

O objeto de resposta terá uma estrutura semelhante ao exemplo a seguir:

```json
[
    {
        "id": 12345678,

>         "user_id": USER-ID,

Reviewer: ```suggestion
        "user_id": "USER_ID",
```

        "begin_date": "2015-05-01T00:00:00Z",
        "end_date": "2015-06-01T23:59:59Z",
        "file_name": "release-report-USER_ID-2016-01-20-131015.csv",
        "created_from": "manual",
        "date_created": "2016-01-20T10:07:53.000-04:00"
    },
    {
        ...
    }
]
```

> ### 3. *Download*

Reviewer: ```suggestion
### 3. Download
```

curl -X POST \
    -H 'accept: application/json' \
    -H 'content-type: application/json' \
    -H 'Authorization: Bearer ENV_ACCESS_TOKEN' \
    'https://api.mercadopago.com/v1/account/release_report/schedule'
```
```php
<?php
include('vendor/rmccue/requests/library/Requests.php');
Requests::register_autoloader();
$headers = array(
    'accept' => 'application/json',
    'content-type' => 'application/json',
    'Authorization' => 'Bearer ENV_ACCESS_TOKEN'
);
$response = Requests::post('https://api.mercadopago.com/v1/account/release_report/schedule', $headers);
```
```java
URL url = new URL("https://api.mercadopago.com/v1/account/release_report/schedule");

HttpsURLConnection connection = (HttpsURLConnection) url.openConnection();

Para parar a geração automática do relatório, atualize o atributo *`scheduled`* na configuração para *`false`*:

System.out.println(connection.getResponseCode());
System.out.println(connection.getResponseMessage());
System.out.println(connection.getInputStream());
```
```python
import requests

headers = {
    'accept': 'application/json',
    'content-type': 'application/json',
    'Authorization': 'Bearer ENV_ACCESS_TOKEN'
}

### 2. Configuração

> De acordo com o cenário desejado, execute os *curls* abaixo para iniciar e cancelar, respectivamente, a geração programada do seu relatório:

Reviewer: ```suggestion
É possível configurar seus relatórios de acordo com o que você precisa. A seguir, mostramos quais as chamadas de API necessárias para iniciar e cancelar a geração programada do seu relatório:
```

#### Iniciar geração automática

> xxxxxxxx

Reviewer: @NaNieto aquí falta texto, puede ser?

> #### Cancelar geração automática

Reviewer: ```suggestion
#### Cancele a geração automática
```

    'Authorization': 'Bearer ENV_ACCESS_TOKEN'
}

response = requests.delete('https://api.mercadopago.com/v1/account/release_report/schedule', headers=headers)
```
```node
var request = require('request');

var headers = {
    'accept': 'application/json',
    'content-type': 'application/json',
    'Authorization': 'Bearer ENV_ACCESS_TOKEN'
};

var options = {
    url: 'https://api.mercadopago.com/v1/account/release_report/schedule',
    method: 'DELETE',
    headers: headers
};

function callback(error, response, body) {
    if (!error && response.statusCode == 200) {
        console.log(body);
    }
}

request(options, callback);
```
]]]

Você receberá um `HTTP STATUS 200 (OK)` em resposta se nenhum erro for encontrado.

O objeto de resposta terá uma estrutura semelhante ao exemplo de cancelamento de relatório a seguir:

```json
{
    "id": 2787882,
    "begin_date": "2019-08-15T06:00:00Z",
    "created_from": "schedule",
    "end_date": "2019-08-16T05:59:59Z",
    "generation_date": "2019-08-16T02:00:00.000-04:00",
    "last_modified": "2019-08-15T15:41:53.681-04:00",
    "report_type": "release",
    "retries": 0,
    "status": "deleted",
    "user_id": USER_ID
}
```

> ### 3. *Download*

Reviewer: ```suggestion
### 3. Download
```
